packages/ibm-products/src/components/ConditionBuilder/ConditionBuilderButton/ConditionBuilderButton.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import cx from 'classnames';
+
+import PropTypes from 'prop-types';
+import { Tooltip } from '@carbon/react';
+import { pkg } from '../../../settings';
+
+const blockClass = `${pkg.prefix}--condition-builder`;
+
+export const ConditionBuilderButton = ({
+  className,
+  label,
+  hideLabel = false,
```

Comment:
as a convention we should continue to declare these defaults in a separate variable

```
const defaults = { ... }
...
hideLabel = defaults.hideLabel
```

Reply:
modified as advised